eslint.config.mjs

```
@@ -42,115 +43,137 @@ export default defineConfig([globalIgnores([
-        }],
+      "no-console": "warn",
+      "react/prop-types": "off",
+      "react/jsx-uses-react": "off",
+      "react/react-in-jsx-scope": "off",
+      "react-hooks/exhaustive-deps": "off",
+      "jsx-a11y/click-events-have-key-events": "warn",
+      "jsx-a11y/interactive-supports-focus": "warn",
+      "jsx-a11y/img-redundant-alt": "off",
+      "@next/next/no-img-element": "off",
+      "prettier/prettier": "off",
+      "no-unused-vars": "off",
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)